README.md

```diff
@@ -8,11 +8,11 @@
 ## About
 
 The Devfile Parser library is a Golang module that:
-1. parses the devfile.yaml as specified by the [api](https://devfile.github.io/devfile/api-reference.html) & [schema](https://github.com/devfile/api/tree/main/schemas/latest).
-2. writes to the devfile.yaml with the updated data.
+1. parses a devfile as specified by the [api](https://devfile.github.io/devfile/api-reference.html) & [schema](https://github.com/devfile/api/tree/main/schemas/latest).
```

Comment:
> https://devfile.github.io/devfile/api-reference.html

The link returns 404. I think it should be updated

Reply:
Good catch. Where is the API reference located now? https://devfile.io/docs/2.2.1/devfile-schema?

Reply:
Fixed in [`44e7c7d` (#186)](https://github.com/devfile/library/pull/186/commits/44e7c7d4c142ba9886bee250397a82f17413fd35)